CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 - Add `Context` object that can be used to build Init/Handle/Migrate response
   via `add_log`, `add_message`, `set_data` and then convert to the proper type
   via `into` or `try_into`. Option to simplify response construction.
+- Env uses `HumanAddr` for `message.sender` and `contract_address`
```

Comment:
So this point is ABI breaking, right? Then we should bump `cosmwasm_vm_version_2` to `cosmwasm_vm_version_3`.

Can you store the new contract as `contract_0.10.wasm`, link `contract.wasm` to `contract_0.10.wasm` and keep the `contract_0.9.wasm` unchanged?

Reply:
In https://github.com/CosmWasm/cosmwasm/pull/491 I add all the basic infrastructure. When you update this PR, you just need to rebuild the 0.10 contract in order to ensure host and guest use the same `Env` type.